.github/workflows/test.yaml

```diff
@@ -22,12 +22,15 @@ jobs:
   integration-test:
     name: Run Integration Tests
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: [14, 16, 18]
```

Comment:
Node 14 is currently in maintenance mode (EOL in April), can either keep it until then or remove it now.

Reply:
We currently support 14 for our regular suite of unit tests, and we should continue to support it till it goes EOL - but we can take April as a chance to do another major version bump!